common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -148,8 +148,8 @@ trait PerformanceSwitches {
 
   val DiscussionSwitch = Switch(
     SwitchGroup.Performance,
-    "discussion",
-    "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
+    "discussion-visible-web",
+    "If this switch is on, comments are displayed on articles on web. This switch only removes the UI from the articles on web and does not affect requests to DAPI.",
```

Comment:
Yep will reduce posts as people on web wont actually see the form but you can still hit all the endpoints. The wording is just emphasising that the endpoints are still open although partially more hidden to users than before.